assets/js/hello-elementor.js

```diff
@@ -0,0 +1,720 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
What is the source of this file? in the `webpack` I see only 2 files: `hello-frontend` & `hello-editor`.

Reply:
Must have been an old file. Removed.